src/device/clint.rs

```diff
@@ -332,9 +332,11 @@ impl VirtClint {
     }
 
     /// Mark the policy MSI as pending for each hart.
-    pub fn set_all_policy_msi(&self) {
+    pub fn set_all_policy_msi(&self, mask: usize) {
```

Comment:
That requires updating the doc comment, and probably the name too to stay consistent.